sciLifeLab_utils/run_QC_analysis.py

```diff
@@ -47,7 +47,14 @@ def main(args):
         flowcells_dirs  = [os.path.join(sample_data_dir,flowcell) \
                 for flowcell in os.listdir(sample_data_dir) \
                 if os.path.isdir(os.path.join(sample_data_dir,flowcell))]
-
+        
+        # to adapt the diretory structure of X runs
+        checked_fc = [fc for fc in flowcells_dirs if re.match(r'^[A-Z]$', os.path.basename(fc))]
```

Comment:
@vezzi : I am also not completely satisfied with this approach, I just did like this cause it would work for both the formats. Like you suggested I will use `glob`, but wondering if I can use it directly to collect the fastq files ?

As we are not using the `flowcell` name itself for anything, can I use something like below and skip the generation of `flowcell_dirs` list at all ?

```
sample_files = glob(os.path.join(sample_data_dir, "*", "*.gz"))
sample_files += glob(os.path.join(sample_data_dir, "*", "*", "*.gz"))
```